packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
+The following software is required to work with SuiteCloud Extension for Visual Studio Code:
+- Node.js version 14.16.0 LTS
+- Oracle JDK version 11
 
-![Account customization project](resources/acpMain.png)
+Read the detailed list of prerequisites in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installation Requirements for SuiteCloud Extension for Visual Studio Code</a>.
 
-SuiteCloud extension for Visual Studio Code is part of SuiteCloud Software Development Kit (SuiteCloud SDK), a set of tools to customize NetSuite accounts.
+## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual Studio Code </font> </span>
+To install SuiteCloud Extension for Visual Studio Code, follow these steps:
 
-💡 To read more about SuiteCloud SDK, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_156026236161.html)
+1. Launch Visual Studio Code and select the **Extensions** tab.
+2. In the search field, enter **SuiteCloud Extension for Visual Studio Code**.
+3. Click **Install**.
+4. Restart Visual Studio Code after installation.
 
-## Installation prerequisites
-The following software is required to work with SuiteCloud extension for Visual Studio Code:
-- Node.js version 14.16.0 LTS
-- Oracle JDK version 11
+See more details about installation in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installing and Setting Up SuiteCloud Extension for Visual Studio Code</a>.
+
+## <font size="6"> <span style="color:gray"> Getting Started </font> </span>
+
+### <font size="5"> <span style="color:gray"> How to Trigger the Command Palette </font>
+Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. The Command Palette allows you to perform several actions in a straightforward way, improving the user experience to the most. To trigger it, use the shortcut, which is Ctrl+Shift+P by default, or click **View** > **Command Palette**. 
+
+<p align="center"><a href="#"><img src="resources/CommandPalette.png" alt="Command Palette" width="500" height="600"></a></p>
 
-Read the detailed list of prerequisites in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html).
+### <font size="5"> <span style="color:gray"> Setting Up an Account </font> </span>
+To start using the SuiteCloud Extension for Visual Studio Code, you first need to set up the NetSuite accounts you want to customize. The account set up includes adding an account to the extension and setting an account as default. 
 
+To add an account, follow these steps:
+1. Click Ctrl+Shift+P to open the Command Palette.
+2. Enter **SuiteCloud** and from the dropdown list, select **SuiteCloud: Set Up Account**.
+3. From the dropdown list, select **New authentication ID**.
```

Comment:
Since there is always a delay before the next dropdown is displayed (it has to load the available authids), would it be better to say something like :
Once the next dropdown list is shown.

Reply:
It is better, but it appears as is in all the Help doc (many occurrences). I will change it in the Readme file and when we do a doc audit will change it in the Help docs.